### PR TITLE
Use garden-rootfs/garden-fuse rootfses

### PR DESCRIPTION
**Pull request by winkingturtle-vmw**
Remove any unmaintained image that we don't build

### CODE REVIEW

**Review by ebroberson — APPROVED** [review]
LGTM

File: security_test.go [hunk 1]
```diff
@@ -236,17 +236,13 @@ var _ = Describe("Security", func() {
...
-			})
-
 			It("executes with correct uid, gid, and supplementary gids", func() {
 				stdout := runForStdout(container, garden.ProcessSpec{
 					User: "alice",
 					Path: "/bin/sh",
 					Args: []string{"-c", "id -u; id -g; id -G"},
 				})
-				Expect(stdout).To(gbytes.Say("1001\n1001\n1001\n"))
+				Expect(stdout).To(gbytes.Say("1001\n1010\n1010 1011\n"))
```
Comment (MarcPaquette) [inline security_test.go:245]: Is there a missing newline here?  The string looks a bit odd.

File: security_test.go [hunk 2]
```diff
@@ -304,7 +300,7 @@ var _ = Describe("Security", func() {
 
 				_, err := container.Run(garden.ProcessSpec{
 					User: "alice",
-					Path: "ifconfig", // ifconfig is only available in /sbin
+					Path: "hello-world", // hello-world is only available in /sbin
```
Comment (MarcPaquette) [inline security_test.go:303]: What caused this to change? Wouldn't `ip` be a better choice?

**Review by MarcPaquette — CHANGES_REQUESTED** [review]
Hey @winkingturtle-vmw 
Added a few questions about these changes

File: networking_test.go [hunk 3]
```diff
@@ -72,7 +72,7 @@ var _ = Describe("Networking", func() {
 
 	Describe("running as a user other than container root", func() {
 		BeforeEach(func() {
-			imageRef.URI = "docker:///cfgarden/preexisting_users"
+			imageRef.URI = "docker:///cloudfoundry/garden-rootfs"
```
[the remark this answers is not in the document]
Comment (winkingturtle-vmw) [reply networking_test.go:75]: Yes. `alice` has always been a user in the container

File: cpu_throttling_test.go [hunk 4]
```diff
@@ -100,7 +100,7 @@ func spinToPunish(container garden.Container, port uint32) {
 }
 
 func startSpinnerApp(container garden.Container, containerPort uint32) {
-	_, err := container.Run(garden.ProcessSpec{Path: "/go/src/app/main"}, garden.ProcessIO{})
+	_, err := container.Run(garden.ProcessSpec{Path: "/bin/throttled-or-not"}, garden.ProcessIO{})
```
[the remark this answers is not in the document]
Comment (winkingturtle-vmw) [reply cpu_throttling_test.go:103]: `main` binary is the same as `throttled-or-not`, it's just renamed

File: security_test.go [hunk 5]
```diff
@@ -236,17 +236,13 @@ var _ = Describe("Security", func() {
 		})
 
 		Context("when running a command as a non-root user", func() {
-			JustBeforeEach(func() {
-				createUser(container, "alice")
-			})
-
```
[the remark this answers is not in the document]
Comment (winkingturtle-vmw) [reply security_test.go:242]: Yes

File: security_test.go (hunk 1 above)
Comment (winkingturtle-vmw) [reply security_test.go:245]: No, alice is just part of two groups. Context: https://github.com/cloudfoundry/wg-app-platform-runtime-ci/blob/main/garden-runc-release/dockerfiles/garden-rootfs/Dockerfile#L81-L82

File: security_test.go (hunk 2 above)
Comment (winkingturtle-vmw) [reply security_test.go:303]: I think this test is not really testing anything, but I decided to leave it as is. The test is testing for a binary that is on the path, in this case, it picks `/sbin`. busbox images had no binary under `/sbin`, so I added [`hello-world` to fulfill this test](https://github.com/cloudfoundry/wg-app-platform-runtime-ci/blob/main/garden-runc-release/dockerfiles/garden-rootfs/Dockerfile#L75-L76), but I think we can remove this test.

**Review by MarcPaquette — APPROVED** [review]

**Pull request merged**